release-notes/v1.4.0.md

```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
 
 |ID|Component |Title|Description|Workaround|
 |--|---------|-----|-----------|----------|
+|CIDG-1928|Volume Groups | Volume group deletion|Volume group can be deleted even though some PVC's are still associated to it | Due to a limitation in volume group sidecar, we are not able to prevent this operation from happening. It's better to check for the `kubectl get events` to understand the volume group deletion failures|
```

Comment:
Can we mention that the volume group CR gets deleted but the a volume group content and the storage backed volume group remains ?